hack/test-go.sh

```diff
@@ -181,7 +181,7 @@ else
             -o -path "${kubernetes_path}/test/e2e"                                                    \
             -o -path "${kubernetes_path}/cmd/libs/go2idl/client-gen/testoutput/testgroup/unversioned" \
           \) -prune                                                                                   \
-        \) -name '*_test.go' -print0 | xargs -0n1 dirname -z | sort -uz | xargs -0n1 printf "${OS_GO_PACKAGE}/%s\n")"
+        \) -name '*_test.go' | xargs -n1 dirname | sort -u | xargs -n1 printf "${OS_GO_PACKAGE}/%s\n")"
```

Comment:
Before your changes in the previous PR, the chain was:
```
-name '*_test.go' -print0 | xargs -0n1 dirname | sort -u | xargs -n1 printf "${OS_GO_PACKAGE}/%s\n
```

find added nils, xargs stripped them. that ensured each path was passed to dirname correctly

once dirname processed it, we were dealing with paths local to the repo, where we don't allow spaces. the nils were in place to guard against spaces in the parent dir structure